lib/Runtime/Library/ArgumentsObject.h

```diff
@@ -122,7 +127,9 @@ namespace Js
         BOOL IsFormalArgument(PropertyId propertyId);
         BOOL IsFormalArgument(PropertyId propertyId, uint32* pIndex);    // Checks whether property is numeric, and on success sets that index.
         BOOL IsArgumentDeleted(uint32 index) const;
+    public:
         const ActivationObject* const GetFrameObject() { return frameObject; }
+        void SetFrameObject(ActivationObject * value) { frameObject = value; }
```

Comment:
Done.

---
In reply to: [68136675](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r68136675) [](ancestors = 68136675,67801971,67771245)